extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -25,7 +26,7 @@ import {
   publicKeySelector,
 } from "background/ducks/session";
 
-const KEY_ID = "keyId";
+let DEFAULT_ACCOUNT_ID = 0;
```

Comment:
WIP: need to track which iteration of the mnemonic the user is on

Reply:
This will likely need to be store in localStorage to keep it from being wiped on extension refresh